src/NServiceBus.Core.Tests/Msmq/MsmqSubscriptionStorageQueueTests.cs

```diff
@@ -232,6 +232,32 @@ public void Messages_with_the_same_timestamp_have_repeatedly_same_order()
             Assert.AreEqual(queue1.Messages.Single(), queue2.Messages.Single());
         }
 
+        [Test]
+        public async Task ShouldIgnoreMessageVersionOnSubscriptions()
```

Comment:
All tests in this file use underscore syntax. Can we at least try to stay consistent inside the same file ;)